README.md

```diff
@@ -0,0 +1,272 @@
+<img src="https://raw.githubusercontent.com/shabados/presenter/dev/resources/icon.png" width="128" alt="Shabad OS">
+
+# Database
+
+A digital representation of Sikh Bani and other Panthic texts with a public logbook of sangat-sourced corrections
+
+[![NPM Version][npm-image]][npm-url]
+[![NPM Downloads][downloads-image]][downloads-url]
+[![Release][release-image]][release-url]
+[![Next Release][next-image]][next-url]
+
+**Table of Contents**
+
+- [Introduction](#introduction)
+  - [Features](#features)
+  - [Screenshots](#screenshots)
+    - [Proofreading Flow [GIF]](#proofreading-flow-gif)
+    - [Database Schema Layout](#database-schema-layout)
+    - [NPM JS package API [GIF]](#npm-js-package-api-gif)
+- [Community](#community)
+- [Contributing](#contributing)
+- [People](#people)
+- [Feedback](#feedback)
+- [Related Projects](#related-projects)
+- [Code of Conduct](#code-of-conduct)
+- [License](#license)
+- [Gurbani and Panthic Compositions](#gurbani-and-panthic-compositions)
+  - [Sri Guru Granth Sahib Ji](#sri-guru-granth-sahib-ji)
+  - [Sri Dasam Granth](#sri-dasam-granth)
+  - [Bhai Gurdaas Ji](#bhai-gurdaas-ji)
+  - [Bhai Nand Laal Ji](#bhai-nand-laal-ji)
+- [Acknowledgments](#acknowledgments)
+
+## Introduction
+
+Shabad OS Database is an open source project with ongoing development, project management, and marketing made possible entirely by the support and dedication of unpaid volunteers.
+
+### Features
+
+- Open-source
+  - Content, history, and modifications are publicly accessible
+  - Transparency is more accountable (i.e. anyone can verify integrity)

```

Comment:
```suggestion
  - More accountable through transparency (i.e. anyone can verify integrity)
```

Reply:
The only reason I wouldn't commit this is because there is a clear pattern between the lines above and below it in the same indent level of "Subject *is* something"

Reply:
Any suggestions for other wording? I don't think it quite makes sense like this

Reply:
```suggestion
  - Public source code is more accountable (i.e. anyone can verify integrity)

```

Reply:
```suggestion
  - Content change history is transparent & accountable (i.e. anyone can verify integrity)
```

Reply:
Isn't that what the first bullet point, the line before this, says?

> Content, history, and modifications are publicly accessible
> Content change history is transparent & accountable (i.e. anyone can verify integrity)

Reply:
Basically trying to say that because this is all publicly accessible, we can be held accountable more transparently/easily

Reply:
```suggestion
  - Transparency through open-source is more accountable (i.e. anyone can verify integrity)
```

Maybe this is better